### PR TITLE
[MINI-5007] Fix Size and Clear Storage Display on first launch

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Show Get Size 0 and Clear storage success on first launch.
Screen Shot 2022-05-09 at 10 35 00 AM

## Links
MINI-5007

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

**Comment by munir-rakuten** [comment]
Resolved with [MINI-5008](https://github.com/rakutentech/android-miniapp/pull/439)

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt [hunk 1]
```diff
@@ -50,13 +50,12 @@ internal class MiniAppSecureStorage(private val activity: Activity) {
...
         miniAppId: String,
         onSuccess: (Long) -> Unit,
-        onFailed: (MiniAppSecureStorageError) -> Unit
     ) {
         if (isStorageAvailable(miniAppId)) {
             val sizeInBytes = File(secureStorageBasePath, "$miniAppId.txt").length()
             onSuccess(sizeInBytes)
         } else {
-            onFailed(MiniAppSecureStorageError.secureStorageUnavailableError)
+            onSuccess(0)
```
Comment (rleojoseph) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt:58]: So the default size is `0`?

**Review by rleojoseph — APPROVED** [review]
Thanks @munir-rakuten  Looks good to me :+1:

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt (hunk 1 above)
Comment (munir-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt:58]: @rleojoseph yes for a non existance file, there is no file there so returning 0 is ok i think.

**Pull request merged**